jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -87,12 +99,15 @@ def load_profiles(self, secret_name="jupyter-singleuser-profiles", filename=None
     self.profiles = []
     self.sizes = []
     if self.api_client:
-      config_map_yaml = self.read_config_map(secret_name, key_name)
-      if config_map_yaml:
-        self.sizes.extend(config_map_yaml.get("sizes", [self.empty_profile()]))
-        self.profiles.extend(config_map_yaml.get("profiles", [self.empty_profile()]))
-      else:
-        _LOGGER.error("Could not find config map %s" % config_map_name)
+      profiles_config_maps = ['secret_name']
```

Comment:
This should be a variable `secret_name` not a string

Reply:
Good catch. This was stupid lol. I'll pull in your commit fixing this.